### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN034.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#81] [General]
# https://blog.3place.cn/3place/lmyn/LMYN034.html

https://blog.3place.cn/3place/lmyn/LMYN034.html

**Comment by picklecai** [comment]
俱舍对国王俱舍那婆，
罗摩呀！这样把话说完，
然后他就飞升到天宫，
走进了那永恒的梵天。 （1.33.4）

天机不可泄露。好像在升天之前，有一次泄露天机的特权。

**Comment by picklecai** [comment]
罗摩呀！他就是我的父亲，
那位非常虔诚的迦亭，
罗摩呀！我叫㤭尸迦，
因为我出自俱舍世系中。 （1.33.6）

没想到这是众友自己家的故事。

**Comment by picklecai** [comment]
罗摩带着罗什曼那，
心里感到非常吃惊，
他向虎般牟尼致敬，
也就沉沉进入梦境。 （1.33.20）

罗摩吃惊的是啥？
